mmdeploy/codebase/mmyolo/models/dense_heads/__init__.py

```diff
@@ -0,0 +1,4 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from .yolov5_head import yolov5_head__predict_by_feat
```

Comment:
use `from . import yolov5_head`. We do not want to expose the rewriter to the module outside.

Reply:
Got it